package.json

```diff
@@ -3,29 +3,29 @@
   "version": "4.15.0",
   "description": "The official documentation of CKEditor 4. https://ckeditor.com/docs/",
   "dependencies": {
-    "@angular-devkit/build-angular": "^0.13.10",
-    "@angular/common": "^7.2.2",
-    "@angular/compiler": "^7.2.2",
-    "@angular/compiler-cli": "^7.2.2",
-    "@angular/core": "^7.2.2",
-    "@angular/forms": "^7.2.2",
-    "@angular/platform-browser": "^7.2.2",
-    "@angular/platform-browser-dynamic": "^7.2.2",
-    "@angular/router": "^7.2.2",
+    "@angular-devkit/build-angular": "^0.1000.4",
+    "@angular/common": "^10.0.6",
+    "@angular/compiler": "^10.0.6",
+    "@angular/compiler-cli": "^10.0.6",
+    "@angular/core": "^10.0.6",
+    "@angular/forms": "^10.0.6",
+    "@angular/platform-browser": "^10.0.6",
+    "@angular/platform-browser-dynamic": "^10.0.6",
+    "@angular/router": "^10.0.6",
     "@babel/core": "^7.1.6",
     "@babel/preset-env": "^7.1.6",
     "@babel/preset-react": "^7.0.0",
-    "@ngtools/webpack": "^7.2.3",
+    "@ngtools/webpack": "^10.1.0",
     "@wiris/mathtype-ckeditor4": "^7.16.0",
     "babel-loader": "^8.0.4",
     "chalk": "^2.4.1",
     "cheerio": "^1.0.0-rc.2",
-    "ckeditor4-angular": "^1.0.0",
+    "ckeditor4-angular": "^1.3.0",
     "ckeditor4-plugin-spreadsheet": "^1.2.0",
-    "ckeditor4-react": "^1.0.0",
-    "ckeditor4-vue": "^0.1.0",
+    "ckeditor4-react": "^1.2.0",
+    "ckeditor4-vue": "^1.1.0",
```

Comment:
Interesting, in my local docs copy I had latest integrations version, while e.g. `^0.1.0` for Vue shouldn't allow it...

Reply:
:point\_up: Just a random thought, nothing to do here :stuck\_out\_tongue:

Reply:
:point\_up: Just a random thought, nothing to do here :stuck\_out\_tongue:

Reply:
Not sure why but also I had `ckeditor4-angular 1.2.2` installed although `^1.0.0` should install version `1.3.0`. Not sure if I misunderstood the idea or something doesn't work well 🤔